subsys/fs/Kconfig.fatfs

```diff
@@ -25,7 +25,8 @@ config FS_FATFS_READ_ONLY
 	  ELM FAT module.
 
 config FS_FATFS_MKFS
-	bool
+	bool "mkfs support for FATFS"
+	default y if FILE_SYSTEM_MKFS
```

Comment:
FAT FS not FATFS; also this makes the `FS_FATFS_MKFS` public Kconfig option so release notes entry required on introduction of the Kconfig as public.

Reply:
> release notes entry required

Where should I add that?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/release-notes-3.7.rst in additional commit.